.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:vue/essential"
```

Comment:
I'd recommend the essential preset. Hope it's not too forgiving for @PVince81 ?!